inc/Engine/WPRocketUninstall.php

```diff
@@ -136,6 +143,24 @@ private function drop_rucss_database_tables() {
 		if ( $this->rucss_usedcss_table->exists() ) {
 			$this->rucss_usedcss_table->uninstall();
 		}
+
+		if ( ! is_multisite() ) {
+			return;
+		}
+
+		foreach ( get_sites( [ 'fields' => 'ids' ] ) as $site_id ) {
+			switch_to_blog( $site_id );
+
+			if ( $this->rucss_resources_table->exists() ) {
+				$this->rucss_resources_table->uninstall();
+			}
+			if ( $this->rucss_usedcss_table->exists() ) {
+				$this->rucss_usedcss_table->uninstall();
+			}
```

Comment:
The duplication here seems wrong.
Can we maybe extract these lines into it's own private function, please.
Maybe the main function is `remove_all_rucss_database_tables()` and the extracted one is `drop_usedcss_and_resources_tables()`